### PR TITLE
Day 1 intro revision 

### PR DESCRIPTION
**Pull request by voetberg**
Comments from Slack: (via Dan Hackett) 

> Okay, first draft of day 1 activity attached (sorry I haven’t figured out the fancy GitHub thing yet). I think in principle this goes from zero to being able to sort of understand what we’ll want to do later in the week. I didn’t like the approach of the existing notebooks, so I started from scratch. Here are some notes on the approach:
> 
> - The intention globally is for the students to do lots of quick “fill in the blank” activities, trying to predict what bits of code will do. It will be much less effective if they don’t try to predict, worth thinking if there’s a better way to emphasize/enforce this
> - The text is minimal and deliberately simple/simplistic. I’ve found long, verbose explanations tend to be too much for group activity settings and get skipped entirely in practice. The intention is they spend most of their time/attention on the code
> - I’d like to try to work more “implicit lessons” into the code examples
> - I didn’t write anything for an ML framework because I only speak Pytorch. If we’re going to use Tensorflow, someone else will need to make that part
> - I didn’t write anything for Pandas because I’m not sure it’s necessary, and I don’t want to terrify them

### CODE REVIEW

File: 01_Introduction/intro.ipynb [hunk 1]
```diff
@@ -0,0 +1,2260 @@
+{
```
Comment (dchackett) [inline 01_Introduction/intro.ipynb:1]: <p>assert x-1 == "????????"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1574968346'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Finish sentence</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038605'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Make less trivial float than 1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038606'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Maybe drop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038607'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Don't use abstract i notation in explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038613'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    print(x) # GUESS:
</pre><p>print(L)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038614'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Add double indexing example L[0][2]</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038615'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>print(type())</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038619'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Go straight to example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038621'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Make comparison to sigma notation explicit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038623'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>sigma notation</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038625'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Move . notation explanation to bit with lists above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038627'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='111'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Not exactly the same as vectors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038628'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='115'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Try example with arrays of differing lengths</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038629'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='120'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Note that this is just making an example array</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038631'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='127'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Negative number slicing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038632'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='133'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>Move up to above</p><p> </p><p>Show reshape failing with bad size</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038636'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='137'/>

Comment (voetberg) [inline 01_Introduction/intro.ipynb:1]: <p>no chatGPT</p><p> </p><p>Add example with wrong shapes failing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/#comment-1575038637'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='163'/>

Comment (dchackett) [reply 01_Introduction/intro.ipynb:1]: <p>I forgot what this means</p><p> </p>

Comment (dchackett) [reply 01_Introduction/intro.ipynb:1]: <p>I forgot what this means</p>

**Pull request merged**